lib/nodejs/reflector.js

```diff
@@ -323,4 +344,75 @@ function OnConnection( socket ) {
     } );
 }
 
-exports.OnConnection = OnConnection;
\ No newline at end of file
+function Evaluate( namespace, node, expression ) {
+
+    return new Promise( function( resolve, reject ) {
+
+        var firstclient = Object.keys( global.instances[ namespace ].clients )[ 0 ];
+        firstclient = global.instances[ namespace ].clients[ firstclient ];
```

Comment:
I think I copied this from somewhere else in the file and didn't clean it up. I'll fix it. Thanks.